app/views/components/_published-dates.html.erb

```diff
@@ -22,7 +22,7 @@
       data-expanded="false"
       data-toggled-text="&#45;&nbsp;<%= t('components.published_dates.hide_all_updates', locale: :en) %>">&#43;&nbsp;<%= t('components.published_dates.show_all_updates', locale: :en) %></a>
       <div class="app-c-published-dates__change-history js-hidden" id="full-history">
-        <ol>
+        <ol class="app-c-published-dates__list">
```

Comment:
If all the class `app-c-published-dates__list` is doing is removing the padding, should that be done with `govuk-!-padding-0`?

Reply:
I think the use of `!important` declarations should be avoided (unless we're looking for a quick-and-dirty fix). This component follows our [convention to use BEM notations](https://github.com/alphagov/govuk_publishing_components/blob/9fc1eae99b6e3eccb8a51dab4458038b7ea63eca/docs/component_conventions.md#bem) with `app-c-published-dates__list` fitting nicely as an element. Today the element has a padding 0, tomorrow may have a border or a background – the markup shouldn't need to be changed to reflect design adjustments.